canopen/objectdictionary/datatypes_24bit.py

```diff
@@ -0,0 +1,33 @@
+import struct
+
+
+class Unsigned24(struct.Struct):
+    def __init__(self, *args, **kwargs):
+        super().__init__("<L", *args, **kwargs)
```

Comment:
I'm suspecting the Struct class uses some clever logic with __new__ instead of __init__. Maybe you can use composition instead of inheritance in this case?

Reply:
thanks for the hint. I'll have another look at it. I used Python 3.10. There the tests succeeded...

Reply:
ok, apparently they changed something between py 3.10 and 3.12, where they enforce that the derived class has the same signature as the base class. I've changed to the composition approach now, as you suggested. The tests succeed now for 3.10 and 3.12. At least locally...